src/api-reference/request/v4.get-started.markdown

```diff
@@ -15,131 +13,88 @@ Version 4.0 of Request API works only with the new [Authentication API](/api-ref
-EMEA Production:  https://emea.api.concursolutions.com/travelrequest/v4/
-China Production: https://cn.api.concurdc.cn/travelrequest/v4/
-```
+## <a name="products-editions"></a>Products and Editions
 
-## <a name="explore-api"></a>Explore the API
+* Concur Request Professional Edition
+* Concur Request Standard Edition
 
-### <a name="prerequisites"></a>Prerequisites
+## <a name="scope-usage"></a>Scope Usage
 
-Read the [Getting Started](/api-reference/authentication/getting-started.html) section of [Authentication API](/api-reference/authentication/apidoc.html).
+Name|Description|Endpoint
+---|---|---
+`travelrequest.write`|Read and write Requests|GET, POST, PUT, DELETE
 
-Once you have registered your application, read about the [API endpoints](/api-reference/request/v4.endpoints.html), or click the button to download a request collection for Postman.
+## <a name="dependencies"></a>Dependencies
 
-[Run in Postman](https://app.getpostman.com/run-collection/bfe85f4a4e435a161a8a)
+SAP Concur clients must purchase Concur Request in order to use this API. This API requires the following additional SAP Concur APIs:
 
-### <a name="token"></a>Retrieve a User Access Token:
+* [List](/api-reference/common/list-item/v3.list-item.html), to retrieve the `listId`
+* [User profile](/api-reference/profile/v1.user.html), to retrieve the `userId`
+* [Expense Report](/api-reference/expense/expense-report/expense-report-get.html), to retrieve the `reportId`
 
-Before issuing requests to the API, you must [obtain an access token from the Authentication API](/api-reference/authentication/getting-started.html).
+## <a name="access-token-usage"></a>Access Token Usage
 
-The response will include an `access_token` field, which contains your access token. For subsequent calls, you will need to include this access token in the `Authorization` header of your calls. An `id_token` will be also included in the response. In order to retrieve the unique ID for your user, you will have to decode this `id_token` at [jwt.io](https://jwt.io/). You will need this ID in order to issue requests on the API.
+This API supports both company level and user level access tokens.
 
-Examples:
+#### Company Access Token
 
-cURL
+* The `userId` parameter is required to provide the user identity of who made the API call.
+* Does not have an associated role.
 
-```shell
-curl -d "client_secret={YOUR SECRET}&client_id={YOUR CLIENT ID}&grant_type=password&username={YOUR USERNAME}&password={YOUR PASSWORD}" https://us.api.concursolutions.com/oauth2/v0/token
-```
+#### User Access Token
 
-HTTPie
+* The `userId` parameter is not required.
+* Requires the Web Service Admin role to call the API.
 
-```shell
-http -f POST https://us.api.concursolutions.com/oauth2/v0/token client_secret={YOUR SECRET} client_id={YOUR CLIENT ID} grant_type=password username={YOUR USERNAME} password=P{YOUR PASSWORD}
-```
+## <a name="explore-api"></a>Explore the API
```

Comment:
Remove this whole section- we don't want to repeat information about authentication in each API. We want to have everyone using the Authentication pages to get that information.

Reply:
@laurentlhermine  idem above. can you please make sure the new file for the Get started page has been taken into account in the PR?

Reply:
done